scripts/styleCheck.fsx

```diff
@@ -166,6 +166,19 @@ let CheckStyleOfXamlFiles(rootDir: DirectoryInfo) : bool =
 
 let rootDir = Path.Combine(__SOURCE_DIRECTORY__, "..") |> DirectoryInfo
 
+// We need this step so we can change the files using `npx prettier --write` in the prettier calls.
+// Otherwise we get permission denied error in the CI.
+Process
+    .Execute(
+        {
+            Command = "chmod"
+            Arguments = "777 --recursive ."
+        },
+        Echo.Off
+    )
+    .UnwrapDefault()
+|> ignore
+
```

Comment:
@realmarv this is not a refactor because you're changing behaviour, you're fixing a bug that I told you, so please squash things properly

Reply:
I squashed this change to the previous commit